### PR TITLE
Database Cleanup: b64 to s3 uri 

### LINKED ISSUE #736: Database Cleanup: Automate stale data removal
**Is your feature request related to a problem?**  
Our database accumulates stale input/outputs, particularly base64 data, which can lead to performance issues and clutter.

**Describe the solution you'd like**  
- Implement a cron job to automatically remove stale input/outputs  
- Focus on base64 data specifically  
- Schedule the job for regular clean-up intervals  

<details><summary>Original issue</summary>

**Describe the current behavior**
A clear description of how it currently works and what the limitations are.

**Describe the enhancement you'd like**
A clear and concise description of the improvement you want to see.

**Why is this enhancement needed?**
Explain the benefits (e.g., performance, usability, maintainability, scalability).

**Additional context**
Add any other context, metrics, screenshots, or examples about the enhancement here.

</details>

### PR DESCRIPTION
**Pull request by Prajna1999**
## Summary

Target issue is #736 
Explain the **motivation** for making this change. What existing problem does the pull request solve?

## Checklist

Before submitting a pull request, please ensure that you mark these task.

- [ ] Ran `fastapi run --reload app/main.py` or `docker compose up` in the repository root and test.
- [ ] If you've fixed a bug or added code that is tested and has test cases.

## Notes

Please add here if any other information is required for the reviewer.

## Release Notes

* **New Features**
  * Added a migration utility to efficiently transfer stored audio data to cloud storage with batch processing, error tracking, skip handling, and progress monitoring.

* **Tests**
  * Added comprehensive test suite validating the migration utility across multiple scenarios including successful transfers, edge cases, upload failures, and storage optimization.

[Review Change Stack](https://app.coderabbit.ai/change-stack/ProjectTech4DevAI/kaapi-backend/pull/816)

### CODE REVIEW

File: backend/app/api/routes/private.py [hunk 1]
```diff
@@ -20,6 +29,93 @@ class PrivateUserCreate(BaseModel):
     is_verified: bool = False
 
 
+@router.post("/migrate/tts-base64-to-s3", include_in_schema=False)
+def migrate_tts_base64_to_s3(session: SessionDep) -> dict:
```
Comment (AkhileshNegi) [inline backend/app/api/routes/private.py:38]: since this is one off endpoint we can add aggressive logging to cover success and intermittent failures to ensure everything ran as expected

**Review by AkhileshNegi — APPROVED** [review]

**Review by nishika26 — APPROVED** [review]
